aiohttp/pytest_plugin.py

```diff
@@ -216,6 +217,21 @@ def loop(loop_factory, fast, loop_debug):  # type: ignore
         yield _loop
 
 
+@pytest.fixture
+def proactor_loop():  # type: ignore
+    if not PY_37:
+        policy = asyncio.get_event_loop_policy()
+        policy._loop_factory = asyncio.ProactorEventLoop  # type: ignore
+    else:
+        policy = asyncio.WindowsProactorEventLoopPolicy()  # type: ignore
+        asyncio.set_event_loop_policy(policy)
+
+    loop_context(policy.new_event_loop)
```

Comment:
Why do you invoke this twice?

Reply:
Sorry, it was by mistake.

Reply:
Did you forget to push?

Reply:
I had to run to class. So I had to defer pushing to later, just made the changes locally, and commented on the reviews. Pushing now :-D